native/cocos/platform/UniversalPlatform.cpp

```diff
@@ -76,9 +76,11 @@ void UniversalPlatform::setHandleDefaultEventCallback(HandleEventCallback cb) {
 }
 
 int32_t UniversalPlatform::run(int argc, const char** argv) {
+#if (CC_PLATFORM != CC_PLATFORM_OPENHARMONY)
     if (cocos_main(argc, argv) != 0) {
         return -1;
     }
+#endif
```

Comment:
UniversalPlatform should not use PLATFORM MACROS , override it in its sub class?